ci-operator/config/stackrox/scanner/stackrox-scanner-master.yaml

```diff
@@ -448,107 +121,12 @@ tests:
       - mount_path: /tmp/secret/stackrox-scanner-secrets
         name: stackrox-scanner-secrets
         namespace: test-credentials
-      dependencies:
-      - env: SCANNER_SLIM_IMAGE
-        name: scanner-slim
-      - env: SCANNER_DB_SLIM_IMAGE
-        name: scanner-db-slim
       from: src
       resources:
         requests:
           cpu: 2000m
           memory: 4000Mi
   timeout: 3h0m0s
-- as: hourly-store-db-dump
-  cron: 0 */3 * * *
-  steps:
-    test:
-    - as: store-db-dump
-      commands: .openshift-ci/dispatch.sh store-db-dump
-      credentials:
-      - mount_path: /tmp/vault/stackrox-scanner-secrets
-        name: stackrox-scanner-secrets
-        namespace: test-credentials
-      from: db-dump
-      resources:
-        requests:
-          cpu: 2000m
-          memory: 4000Mi
-- as: hourly-store-genesis-dump
-  cron: 0 */3 * * *
-  steps:
-    test:
-    - as: store-genesis-dump
-      commands: .openshift-ci/dispatch.sh store-genesis-dump
-      credentials:
-      - mount_path: /tmp/vault/stackrox-scanner-secrets
-        name: stackrox-scanner-secrets
-        namespace: test-credentials
-      from: genesis-dump
-      resources:
-        requests:
-          cpu: 2000m
-          memory: 4000Mi
-- as: hourly-style-checks
-  cron: 0 */3 * * *
-  steps:
-    test:
-    - ref: stackrox-scanner
-- as: hourly-unit-tests
-  cron: 0 */3 * * *
-  steps:
-    test:
-    - ref: stackrox-scanner
-- as: hourly-upload-db-dump
-  cron: 0 */3 * * *
-  steps:
-    test:
-    - as: upload-db-dump
-      commands: .openshift-ci/dispatch.sh upload-db-dump
-      credentials:
-      - mount_path: /tmp/vault/stackrox-scanner-secrets
-        name: stackrox-scanner-secrets
-        namespace: test-credentials
-      from: db-dump
-      resources:
-        requests:
-          cpu: 2000m
-          memory: 4000Mi
-- as: hourly-upload-dumps-for-downstream
-  cron: 0 */3 * * *
-  steps:
-    test:
-    - as: upload-dumps
-      commands: .openshift-ci/dispatch.sh upload-dumps-for-downstream
-      credentials:
-      - mount_path: /tmp/vault/stackrox-scanner-secrets
-        name: stackrox-scanner-secrets
-        namespace: test-credentials
-      from: db-dump
-      resources:
-        requests:
-          cpu: 2000m
-          memory: 4000Mi
-- as: hourly-upload-dumps-for-embedding
-  cron: 0 */3 * * *
-  steps:
-    test:
-    - as: upload-dumps
-      commands: .openshift-ci/dispatch.sh upload-dumps-for-embedding
-      credentials:
-      - mount_path: /tmp/vault/stackrox-scanner-secrets
-        name: stackrox-scanner-secrets
-        namespace: test-credentials
-      from: genesis-dump
-      resources:
-        requests:
-          cpu: 2000m
-          memory: 4000Mi
-- as: sanity-check-vuln-updates
```

Comment:
Was this migrated?

Reply:
It should've been moved in this commit: https://github.com/stackrox/scanner/pull/1347/commits/c0eececb77b2f046abdf6f2d515ff468981488ff